systemverilog-plugin/UhdmAst.cc

```diff
@@ -1129,13 +1130,13 @@ static void simplify(AST::AstNode *current_node, AST::AstNode *parent_node)
         AST_INTERNAL::current_scope[current_node->str] = current_node;
         break;
     case AST::AST_WIRE:
-        delete_attribute(current_node, UhdmAst::is_simplified_wire());
-        current_node->attributes[UhdmAst::is_simplified_wire()] = AST::AstNode::mkconst_int(1, true);
-        [[fallthrough]];
     case AST::AST_PARAMETER:
     case AST::AST_LOCALPARAM:
-        AST_INTERNAL::current_scope[current_node->str] = current_node;
-        convert_packed_unpacked_range(current_node);
+        if (!current_node->attributes.count(UhdmAst::is_simplified_wire())) {
+            current_node->attributes[UhdmAst::is_simplified_wire()] = AST::AstNode::mkconst_int(1, true);
```

Comment:
Ah, now I see `set_attribute` isn't on master yet. Sorry!

Reply:
Unfortunately ``set_attribute`` is added in PR: https://github.com/chipsalliance/yosys-f4pga-plugins/pull/454 that is not yet merged.

When it will be merged, I can use it.